cmd/user-cluster-controller-manager/main.go

```diff
@@ -162,9 +166,12 @@ func main() {
 	if err != nil {
 		log.Fatalw("Failed parsing clusterURL", zap.Error(err))
 	}
-	if runOp.openvpnServerPort == 0 {
+	if !runOp.isKonnectivityEnabled && runOp.openvpnServerPort == 0 {
 		log.Fatal("-openvpn-server-port must be set")
 	}
```

Comment:
Did you mean `runOp.konnectivityServerPort` here?

Reply:
Nope, `konnectivityServerPort` is defaulted to 6443 if not set.
I just wanted to NOT check for `openvpnServerPort` if konnectivity is enabled.